reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+both Windows PowerShell and PowerShell 7. If your migrating from PowerShell Core 6.x to PowerShell
+7, the installation is upgraded and the path replaced.
+
+PowerShell 7 installation path is added to Microsoft Windows:
+
+- Windows PowerShell 5.1 : `C:\Windows\System32\WindowsPowerShell\v1.0` will add
+`%programfiles%\PowerShell\7`
+
+- PowerShell Core 6.x on Windows: `%programfiles%\PowerShell\6` is upgraded to
+ `%programfiles%\PowerShell\7`
+
+In Windows PowerShell, the executable to launch PowerShell is named `powershell.exe`. In version 6
+and above, the executable is changed to support side-by-side execution. The new executable to launch
+PowerShell 7 is `pwsh.exe`. Preview builds will remain in-place as `pwsh-preview` instead of `pwsh`
+under the 7-preview directory.
+
+## Modules and Module paths
+
+PowerShell 7 on Microsoft Windows combines the Windows PowerShell Module paths and the new
+PowerShell 7 paths to provide autoloading of both Core and Desktop modules. By default, Windows
+PowerShell and PowerShell 7 have different `$Env:PSModulePath` values except for
+`%WINDIR%\system32\WindowsPowerShell\v1.0\Modules` module path which is shared by both.
+
+`$Env:PSModulePath` stores the paths to the default module directories. When importing a module,
+PowerShell checks the specified directories when a full path to the module is not specified. When
+starting a version of PowerShell, the `$Env:PSModulePath` reflects the module search path and only
+includes path segments appropriate for the version of PowerShell being started.
+
+> [!NOTE]
+> PowerShell sets the value of `$PSHOME\Modules` in the registry. It sets the value of
+> `$HOME\Documents\PowerShell\Modules` each time you start PowerShell.
+
+The default value of Windows PowerShell `$Env:PSModulePath` on Windows is:
+
+```powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\WindowsPowerShell\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+The default value of PowerShell 7 `$Env:PSModulePath` on Windows combines the Windows PowerShell
+paths and the PowerShell 7 paths to provide autoloading of modules:
+
+```powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\PowerShell\Modules
+C:\Program Files\PowerShell\Modules
+C:\Program Files\PowerShell\7\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+> [!NOTE]
+> Additional paths may exist if the user has added custom modules or applications that
+> impact the path. You will notice additional paths if a user has modified the path through the
+> registry or **Environment Variables** in **System Settings**.
+
+For more information, see `PSModulePath` in [about_Environment_Variables](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7)
```

Comment:
You can deep link it to https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7#environment-variables-that-store-preferences